docs/variables.md

```diff
@@ -754,6 +791,20 @@ not.
 
 Specifies the name of the column family to compact.
 
+### `rocksdb_compact_lzero_now`
+
+| Option       | Description          |
+|--------------|----------------------|
+| Command-line | --rocksdb-compact-lzero-now |
+| Dynamic      | Yes                  |
+| Scope        | Global               |
+| Data type    | Boolean              |
+| Default      | OFF                  |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable acts as a switch. Set the variable to `ON`, `rocksdb-compact-lzero-now=ON`, to immediately compact all the `Level 0` (L0) files. After all the `L0` files are compacted, the variable value automatically switches to `OFF`. The default value is `OFF`, which means the variable is disabled.
```

Comment:
This should be better: This variable acts as a trigger. 
Please remove: The default value is `OFF`, which means the variable is disabled.